CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ This file contains notable changes (e.g. breaking changes, major changes, etc.)
 
 This file was introduced starting Kani 0.23.0, so it only contains changes from version 0.23.0 onwards.
 
+## [0.43.0]
+
+## What's Changed
+* Make `codegen_span` stable by @ouz-a in https://github.com/model-checking/kani/pull/2882
+* Change reachability module to use StableMIR by @celinval in https://github.com/model-checking/kani/pull/2894
+* Fix StableMIR migration documentation by @celinval in https://github.com/model-checking/kani/pull/2915
+* Stabilize `find_debug_info` by @ouz-a in https://github.com/model-checking/kani/pull/2912
+* Migrate hooks and place modules to use mostly StableMIR APIs by @celinval in https://github.com/model-checking/kani/pull/2910
+* Migrate static handling and most of the operand codegen code to StableMIR by @celinval in https://github.com/model-checking/kani/pull/2931
+* Migrate `rvalue` and `coercion` module to use StableMIR by @celinval in https://github.com/model-checking/kani/pull/2938
+* Migrate `intrinsics` module to use StableMIR by @celinval in https://github.com/model-checking/kani/pull/2939
+
```

Comment:
Missing the toolchain update

Reply:
I added the non-automatic toolchain updates.